guides/dcos-microservices/src/main/paradox/index.md

```diff
@@ -2,4 +2,151 @@
\ No newline at end of file
+You've created your beautiful microservice system using [Lagom](http://www.lagomframework.com/) and you now need to deploy it in its entirety to DC/OS. There are many moving parts to a microservices system including service discovery and event sourcing; both of which assume a distributed system. Lagom is based on [Play](https://www.playframework.com/) and [Akka](http://akka.io/) and they are Highly Available (HA) by default, thereby requiring clustering. So, how then do you deploy these necessarily complex systems to DC/OS?
+
+## The solution
+
+What we'll do in this guide is use [ConductR](http://conductr.lightbend.com/) to deploy your Lagom service. ConductR 2.0 can operate either in a stand-alone manner or as a Mesos framework\*. ConductR has been designed from the ground-up with Mesos in mind, and represents the easiest way to manage Reactive Platform based applications and services on Mesos platforms, including DC/OS. For example, one thing you do not have to think about managing is establishing Akka cluster systems, and another is service discovery. You'll get both of these important and powerful capabilities taken care of for you. 
+
+> As at the time of writing ConductR 2.0 is available as an Early Access release and can be obtained by [contacting our sales department](https://www.lightbend.com/contact). We will also be offering a free version of ConductR in the near future and will update this guide for when it becomes available. Meanwhile read on to gain a feel for how straightforward it is to deploy your Lagom system to DC/OS!
+
+\* _Note that Mesos frameworks are also known as "services" to DC/OS._ 
+
+### Prerequisites
+
+The solution proposed here utilizes [sbt](http://www.scala-sbt.org/) and a sample Lagom system known as "Chirper". The following are prerequisites to the solution:
+
+* an [installation of ConductR on DC/OS](http://conductr.lightbend.com/docs/2.0.x/Install#DC/OS-Installation) with connectivity to it via the `dcos` command line tool; 
+* a working instance of Cassandra hosted by DC/OS;
+* having installed the [ConductR CLI](https://github.com/typesafehub/conductr-cli#command-line-interface-cli-for-lightbend-conductr) and run the `conduct setup-dcos` command in order to be able to use `dcos conduct` commands; and
+* the cloning of the [Lagom Chirper repository](https://github.com/lagom/activator-lagom-java-chirper).
+
+### Our approach
+
+We will generate an installation script representing our deployment and then tailor it for DC/OS. This will include:
+
+* removing the Cassandra deployment from the installation script given that DC/OS will be hosting Cassandra; and
```

Comment:
Would be nice to add an option to the installation script to not include Cassandra into it.